tools/train_net.py

```diff
@@ -134,6 +148,8 @@ def main():
             # data
             start_time = time.time()
             data = next(data_loader_iter)
+            if distributed:
```

Comment:
I think DDP already has the function of sending data to the specific device. Please correct me if I'm wrong. Do we 
 only need to do this when `distributed` is False?

Reply:
that's right! That's why I corrected with """distributed is False"""